bigquery_etl/view/generate_stable_views.py

```diff
@@ -0,0 +1,251 @@
+"""
```

Comment:
What do you think about adding this to the bqetl cli? See #1648

Reply:
I will file that as an additional follow-up issue to this PR. This script will be used by ops logic and for now I'm attempting to maintain as similar an interface as possible to the existing `generate_views` script.